ping_identity/README.md

```diff
@@ -0,0 +1,52 @@
+# Ping Identity - Datadog API Connector
+
+## Overview
+
+Ping Identity delivers intelligent identity solutions for the enterprise. We enable companies to achieve Zero Trust identity-defined security and more personalized, streamlined user experiences.
+
+Datadog integrates directly with the PingOne DaVinci platform through the Datadog API connector. PingOne DaVinci is an orchestration platform that lets you create flows using connections and logical operators. These flows guide users through defined processes that can present customized pages, modify values, or perform other actions. 
```

Comment:
A few initial comments:

Do you have a more concrete example of what a user could build with DaVinci?
Can you also add more color into the kinds of actions users can perform with Datadog in DaVinci?